postman/AskDarcel%20API.postman_collection.json

```diff
@@ -444,6 +465,44 @@
 				"description": ""
 			},
 			"response": []
+		},
+		{
+			"name": "Submit New Service Request",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"type": "text/javascript",
+						"exec": [
+							"tests[\"Response time is less than 1000ms\"] = responseTime < 1000;",
+							"",
+							"tests[\"Status code is 201\"] = responseCode.code === 202;"
+						]
+					}
+				}
+			],
+			"request": {
+				"url": "{{base_url}}/resources/1/services",
+				"method": "POST",
+				"header": [
+					{
+						"key": "Accept",
+						"value": "application/json",
+						"description": ""
+					},
+					{
+						"key": "Content-Type",
+						"value": "application/json",
+						"description": ""
+					}
+				],
+				"body": {
+					"mode": "raw",
+					"raw": "{\n    \"rating\": {\n    \t\"rating\": {\n    \t\"user\": {\n    \t\t\"id\": 1\n    \t},\n    \t\"rating\": 3\n    }\n    }\n}"
```

Comment:
@lgarofalo @achadha, what's the correct POST body for creating a new service request? The one in the Postman JSON doesn't look right:

```json
{
    "rating": {
        "rating": {
            "user": {
                "id": 1
            },
            "rating": 3
        }
    }
}
```